src/app.py

```diff
@@ -45,24 +57,71 @@ def gather_articles():
         result = db.articles.bulk_write(article_upserts, ordered=False).upserted_ids
         # Need to unwrap ObjectID objects from MongoDB into str ids
         article_ids = [str(article) for article in result.values()]
-        try:
-            logging.info(f"Sending notification for {len(article_ids)} articles")
-            requests.post(
-                ("http://" + VOLUME_NOTIFICATIONS_ENDPOINT), data={"articleIDs": article_ids}
-            )
-        except:
-            logging.error("Unable to connect to volume-backend.")
+        if article_ids:
+            try:
+                logging.info(f"Sending notification for {len(article_ids)} articles")
+                requests.post(
+                    VOLUME_NOTIFICATIONS_ENDPOINT + "/articles/",
+                    data={"articleIDs": article_ids},
+                )
+            except Exception as e:
+                logging.error("Articles unable to connect to volume-backend.")
+
+
+# Function for gathering magazines for running with scheduler
+def gather_magazines():
+    logging.info("Gathering magazines")
+    magazines = []
+    with MongoClient(MONGO_ADDRESS) as client:
+        db = client[DATABASE]
+        data = sheet.get_all_values()
+        len_data = len(data)
+        parse_counter = 1
+        for i in range(1, len_data):
+            if data[i][5] != "1" and data[i][0] != "":
```

Comment:
A quick comment about what exactly this checks for would be useful; I understand it's checking values in the columns of a particular entry in the sheets but would be useful to document what _exactly_ it's checking for